python/desc/slrealizer/plotting.py

```diff
@@ -5,17 +5,14 @@
 import matplotlib
 import math
 
-def plot_lens_random_date(self, lensID = 7176527, convolve=False):
-    import random
-    randomIndex = random.randint(0, 200)
-    if(self.observation[randomIndex][1]!='y'):
-        currLens = self.lens.get_lens(lensID)
-        drawPlot(self.observation[randomIndex], currLens, convolve)
-    else:
-        # recursive call
-        plot_lens_random_date(self, lensID, convolve)
-
 def determineAlpha(mag_ratio):
+    """
+    Bug: No docstring
+    Bug: Each image should have its own alpha, because the different
+         images have different magnifications (these are in the OM10 "MAG" columns, and need to be applied to each image's sourceX
+         magnitude.)
+    Bug: function name should be "determine_alpha" to be PEP8 compliant
```

Comment:
There's a lot more places where we need to become PEP8 compliant... You should Google this and read about it, the PEP8 standard makes code more readable, and its good practice to adopt it.